package.json

```diff
@@ -1,30 +1,52 @@
 {
   "name": "fast-bmp",
+  "type": "module",
   "version": "2.0.1",
   "description": "A bmp image encoder",
-  "main": "src/index.js",
+  "main": "./lib/index.js",
+  "module": "./lib-esm/index.js",
+  "types": "./lib/index.d.ts",
   "repository": "git@github.com:image-js/fast-bmp.git",
   "author": "Daniel Kostro <kostro.d@gmail.com>",
   "license": "MIT",
+  "files": [
+    "src",
+    "lib",
+    "lib-cjs"
+  ],
   "dependencies": {
-    "iobuffer": "^5.1.0"
+    "iobuffer": "^5.4.0"
   },
   "devDependencies": {
-    "@jest/globals": "^29.1.1",
-    "eslint": "^8.24.0",
-    "eslint-config-cheminfo": "^8.0.2",
+    "@babel/plugin-transform-modules-commonjs": "^7.26.3",
+    "@babel/preset-typescript": "^7.26.0",
+    "@eslint/eslintrc": "^3.3.0",
+    "@eslint/js": "^9.21.0",
+    "@jest/globals": "^29.7.0",
+    "@types/jest": "^29.5.14",
+    "eslint": "^9.21.0",
+    "eslint-config-cheminfo-typescript": "^17.0.0",
+    "globals": "^16.0.0",
     "jest": "^29.1.1",
-    "prettier": "^2.7.1"
+    "prettier": "^2.7.1",
+    "rimraf": "^6.0.1",
+    "typescript": "^5.8.2"
   },
   "scripts": {
     "eslint": "eslint src",
     "eslint-fix": "eslint --fix src",
     "prettier": "prettier --check src",
     "prettier-write": "prettier --write src",
-    "test": "npm run test-only && npm run eslint && npm run prettier",
+    "test": "npm run test-only && npm run eslint && npm run prettier && npm run check-types",
     "test-only": "jest --coverage",
     "test-watch": "jest --watch",
-    "test-write": "FAST_BMP_WRITE_DATA_FILES=1 npm run test-jest"
+    "test-write": "FAST_BMP_WRITE_DATA_FILES=1 npm run test-jest",
```

Comment:
It was already wrong, but there's not test-jest script. I guess it should be test-only.